great_expectations/self_check/util.py

```diff
@@ -2211,11 +2211,12 @@ def check_json_test_result(test, result, data_asset=None) -> None:
                             rtol=RTOL,
                             atol=ATOL,
                         ), f"(RTOL={RTOL}, ATOL={ATOL}) {result['success']} not np.allclose to {value}"
-
                     except TypeError:
                         assert (
                             result["success"] == value
                         ), f"{result['success']} != {value}"
+                else:
+                    assert result["success"] == value, f"{result['success']} != {value}"
```

Comment:
We had stopped testing the `success` status in the `out` segment of the data definition json files since they are all booleans in `great_expectations/tests/test_definitions/*/.json`. This adds it back and happily no tests broke. For a discussion see: https://superconductivedata.slack.com/archives/C026YSF2JA3/p1657581590975119